v-next/hardhat/test/internal/builtin-plugins/network-manager/hook-handlers/config.ts

```diff
@@ -604,6 +604,399 @@ describe("network-manager/hook-handlers/config", () => {
         );
       });
     });
+
```

Comment:
These tests are ported from [this file](https://github.com/NomicFoundation/hardhat/blob/2f5b7a8be71f2183c2992383bcc582bc4c02626b/packages/hardhat-core/test/internal/core/config/config-validation.ts)

I just grouped them in a different way to make the tests more readable